src/physics/cam/wv_saturation.F90

```diff
@@ -259,41 +272,41 @@ end subroutine wv_sat_final
 !---------------------------------------------------------------------
 
 ! Compute saturation vapor pressure over water
-elemental function svp_water(t) result(es)
+function svp_water(t) result(es)
 
   use wv_sat_methods, only: &
        wv_sat_svp_water
 
-  real(r8), intent(in) :: t ! Temperature (K)
-  real(r8) :: es            ! SVP (Pa)
+  real(r8), intent(in)  :: t  ! Temperature (K)
+  real(r8)              :: es ! SVP (Pa)
 
-  es = wv_sat_svp_water(T)
+  es = wv_sat_svp_water(t)
 
 end function svp_water
 
 ! Compute saturation vapor pressure over ice
-elemental function svp_ice(t) result(es)
+function svp_ice(t) result(es)
 
   use wv_sat_methods, only: &
        wv_sat_svp_ice
 
-  real(r8), intent(in) :: t ! Temperature (K)
-  real(r8) :: es            ! SVP (Pa)
+  real(r8), intent(in)  :: t  ! Temperature (K)
+  real(r8)              :: es ! SVP (Pa)
 
-  es = wv_sat_svp_ice(T)
+  es = wv_sat_svp_ice(t)
 
 end function svp_ice
 
 ! Compute saturation vapor pressure with an ice-water transition
-elemental function svp_trans(t) result(es)
+function svp_trans(t) result(es)
```

Comment:
Create a vectorized version of svp_trans

Reply:
Thanks, it is added.